style.css

```diff
@@ -0,0 +1,123 @@
+
+.container {
```

Comment:
[ ] kindly fix your website's margins, Minimum mobile screen size should be `375`px.
The Figma design : 
![design](https://user-images.githubusercontent.com/62156607/193101635-490c1f67-ba58-458a-aa60-65bb476ed737.PNG)

Your design : 
![designta3ou](https://user-images.githubusercontent.com/62156607/193101696-bd3a0ac0-dcdc-4716-a349-561779e5647f.PNG)